packages/ai/src/core.ts

```diff
@@ -111,4 +111,52 @@ export class EdgeDBAI {
 
     return data.response;
   }
+
+  async *getRagAsyncGenerator(
```

Comment:
I don't understand what this method is meant to do? Shouldn't it parse the events out of the sse response stream and have the iterator return typed event objects? Currently it seems like it just outputs whatever random chunks of data we get from the server.

Reply:
> Shouldn't it parse the events out of the sse response stream and have the iterator return typed event objects?

Yeah, that's what we need